samples/Common.c

```diff
@@ -1063,13 +1063,16 @@ STATUS signalingMessageReceived(UINT64 customData, PReceivedSignalingMessage pRe
 
     switch (pReceivedSignalingMessage->signalingMessage.messageType) {
         case SIGNALING_MESSAGE_TYPE_OFFER:
+            // Check if we already have an ongoing master session with the same peer
+            CHK_ERR(!peerConnectionFound, STATUS_INVALID_OPERATION, "Peer connection %s is in progress", pReceivedSignalingMessage->signalingMessage.peerClientId);
```

Comment:
We encounter a issue, If we send a OFFER in the middle of a call to renegotiation media info, for example:  mute/unmute,  the OFFER will be ignored from this line,  How can we support renegotiation ?